recipes/cute_headers/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class CuteHeadersConan(ConanFile):
+    name = "cute_headers"
+    description = "Various single-file cross-platform C/C++ headers implementing self-contained libraries."
+    topics = ("conan", "various", "pure-c")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/RandyGaul/cute_headers"
+    license = "Unlicense"  # This library (cute-headers) is public domain software, i.e. not copyrighted
+    no_copy_source = True
+
+    def _extract_license(self):
+        file = os.path.join(self.package_folder, "include/cute_math2d.h")
+        with open(file, 'r') as f:
+            file_content = f.read()
```

Comment:
https://docs.conan.io/en/latest/reference/tools.html#tools-load

```suggestion
        file_content = tools.load(file)
```